lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -661,6 +703,7 @@ data DefaultFieldValues = DefaultFieldValues
     , defaultDesiredNumberOfPool :: Word16
     , defaultMinimumUTxOValue :: W.Coin
     , defaultHardforkEpoch :: Maybe W.EpochNo
+    , defaultScriptPoolGap :: Maybe Seq.AddressPoolGap
```

Comment:
Why a `Maybe` but more importantly, I don't think we want to add this one here. The fields for which we may want to tweak default values are typically network or genesis parameters, and fields for which we aren't in full control of. For the pool gap, we as wallet decides fully of its value and there's little (i.e. no) chances that we'll want different values for different instances of cardano-wallet. 

I suggest sticking the default value directly in the migration itself.

Reply:
donein https://github.com/input-output-hk/cardano-wallet/pull/2308/commits/3f6a16584b53c5f34581f7012ac5bbff69c5f126